src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/model/Case.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.ons.ctp.integration.contactcentresvc.model;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "caze")
+public class Case {
+  @Id private UUID caseId;
+
+  @Column(name = "case_ref")
```

Comment:
Minor, but is the 'case_ref' name required here?
Just concerned about it being copy+pasted so that half of the fields have this notation and half don't. 
Also can't but help wonder about the column name of 'caseType' which doesn't specify a name.
Update: caseType maps to a 'case_type' column.

Reply:
removed all Column annotations . not needed.